toolchain/check/cpp_overload_resolution.h

```diff
@@ -25,8 +25,9 @@ namespace Carbon::Check {
 // remain valid.
 auto PerformCppOverloadResolution(Context& context, SemIR::LocId loc_id,
                                   SemIR::CppOverloadSetId overload_set_id,
+                                  SemIR::InstId self_id,
                                   llvm::ArrayRef<SemIR::InstId> arg_ids)
-    -> std::optional<SemIR::InstId>;
+    -> SemIR::InstId;
```

Comment:
The comment needs to be updated, since this can no longer return `nullopt`.

Reply:
Done.